res/controllers/Denon-MC7000-scripts.js

```diff
@@ -744,25 +845,25 @@ MC7000.sortLibrary = function(channel, control, value) {
 /* LEDs for VuMeter */
 // VuMeters only for Channel 1-4 / Master is on Hardware
 MC7000.VuMeter = function(value, group) {
-    var VULevelOutValue = engine.getValue(group, "PeakIndicator") ? MC7000.VuMeterLEDPeakValue : value*value*value*value*0x69,
+    var VULevelOutValue = engine.getValue(group, "PeakIndicator") ? MC7000.VuMeterLEDPeakValue : Math.pow(value, 4) * MC7000.VuMeterLEDPeakValue - 1,
```

Comment:
Please use variable names starting with lowercase character, these look like a class.

Reply:
Crazy. In my previous PR this variable was suggested exactly like this by one of your team mates. Now if somebody else reviews then it is no good anymore...